client/src/annoMatrix/annoMatrix.ts

```diff
@@ -597,23 +596,25 @@ Return cache keys for columns associated with this query.  May return
   To be effective, the GC callback needs to be invoked from the undo/redo code,
   as much of the cache is pinned by that data structure.
   */
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'field' implicitly has an 'any' type.
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  _gcField(field, isHot, pinnedColumns) {
+  _gcField(
+    field: Field,
+    isHot: boolean,
+    pinnedColumns: string[] | number[] | [undefined]
```

Comment:
I don't think this should ever get `[undefined]`...which may help with the TS errors below

Reply:
Updated [annoMatrix.ts::_gcField](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L617)